Automattic/newspack-custom-content-migrator
# Town News: Try to avoid using email as display name for authors

I noticed that we can risk creating slugs with emails. This does some string juggling to avoid that.

## How to test
- Run the migration and verify that guest users are created with whatever is before the '@' if there is only an email on them.
